ospf6d/ospf6_lsa.h

```diff
@@ -18,6 +18,7 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
  */
 
+#include "lib/json.h"
 #ifndef OSPF6_LSA_H
 #define OSPF6_LSA_H
```

Comment:
Place the #include "lib/json.h" after #define OSPF6_LSA_H

Reply:
Done